.gitignore

```diff
@@ -0,0 +1,4 @@
+.Rproj.user
+.Rhistory
+.RData
+.Ruserdata
```

Comment:
**[Question, non-blocking]** Is there any R code in this repo that requires these files?